### ISSUE TITLE
[Bug]: Embeddings Cache & Langfuse

### ISSUE THREAD

**Issue by 7flash** [BerriAI/litellm#2055]
### What happened?

When I run litellm configured with embeddings model and langfuse logging and then try to generate embeddings, I receive following error, because its trying to log generated embedding into langfuse, but the response for embedding does not have choices field and it actually doesn't need to be logged, I think it should be ignored by langfuse integration.

```
Traceback (most recent call last):
  File "/Users/gur/miniconda3/envs/jan25/lib/python3.10/site-packages/litellm/integrations/langfuse.py", line 67, in log_event
    output = response_obj["choices"][0]["message"].json()
  File "/Users/gur/miniconda3/envs/jan25/lib/python3.10/site-packages/litellm/utils.py", line 532, in __getitem__
    return getattr(self, key)
  File "/Users/gur/miniconda3/envs/jan25/lib/python3.10/site-packages/pydantic/main.py", line 753, in __getattr__
    raise AttributeError(f'{type(self).__name__!r} object has no attribute {item!r}') from exc
AttributeError: 'EmbeddingResponse' object has no attribute 'choices'
```

**Comment by 7flash** [comment]
It seems like langfuse integration already handles this case and probably the error happens because of incorrect if-statement condition (or embeddings response corresponds to ModelResponse but should be EmbeddingResponse)


https://github.com/BerriAI/litellm/blob/58fb6b2b5924041880b9aca1a7f653653a3527a4/litellm/integrations/langfuse.py#L107

**Comment by ishaan-jaff** [comment]
@7flash, what version of litellm are you running ? I recently added embeddings support

**Comment by ghost** [comment]
>   File "/Users/gur/miniconda3/envs/jan25/lib/python3.10/site-packages/litellm/integrations/langfuse.py", line 67, in log_event
    output = response_obj["choices"][0]["message"].json()

@7flash that line looks out of date, since line 67 now is different. Can you upgrade and let me know if this persists?

**Comment by 7flash** [comment]
I think it was fixed in #2307

now it just shows

WARNING:langfuse:Item exceeds size limit (size: 1517595), dropping item.
